tools/pflua-compile

```diff
@@ -40,12 +40,18 @@ if (not(flags["--bpf-asm"] or flags["--bpf-lua"] or flags["--lua"])) then
    flags["--lua"] = true
 end
 
+
+local filter disable_optimization = false
+if flags["-O"] then disable_optimization = true end
```

Comment:
I'm not fond of using -O here either; I used it to mimic tcpdump. Tcpdump's man page says: "-O     Do not run the packet-matching code optimizer.  This is useful only if you suspect a bug in the optimizer."
What flag would you prefer?